docs/reference/composites.rst

```diff
@@ -326,3 +326,36 @@ Methods
    ReverseAdvanceComposite.sample
    ReverseAdvanceComposite.sample_ising
    ReverseAdvanceComposite.sample_qubo
+
+
+Linear Bias
+===========
+
+Composites for implementing linear biases in D-Wave systems through auxiliary methods. 
```

Comment:
```suggestion
Composite for using auxiliary qubits to bias problem qubits. 
```

As a user I would not know what "auxiliary methods" mean without looking more deeply into the new composite so I think the suggested phrasing will help someone scanning the page understand what it does. I also (1) switched to singular, if we ever add a second composite of this type we can update to plural (2) use "qubit" to signal QPU (in the past "D-Wave system" was synonymous with QPU but with hybrid solvers that's more ambiguous and we use quantum computers or QPU solvers instead).

Reply:
I chose that wording in case we want to do make a composite that uses flux biases as h fields, both would be auxiliary methods for implementing/emulating Linear Biases.

I'm ok with going with your suggestion. If you think that leaving it more general so that we can add the flux bias composite is also ok, then I can signal QPU with

```
Linear Bias
===========

Composites for implementing linear biases in D-Wave QPUs through auxiliary methods. 
```